index.js

```diff
@@ -42,27 +47,34 @@ const parseEvent = (event) => {
 };
 
 // Prepare the Messages and Options
-const prepareLogs = (eventData) => {
+const prepareLogs = (eventData, raw_event_message) => {
     return eventData.logEvents.map((event) => {
+        const eventMetadata = {
+            source: 'cloudwatch'
+            , event: {
+                type: eventData.messageType
+                , id: event.id
+            }, log: {
+                group: eventData.logGroup
+                , stream: eventData.logStream
+            }
+        };
+
+        const line = raw_event_message ? event.message : JSON.stringify(Object.assign({}, {
+            message: event.message
+        }, eventMetadata));
+
+        let meta = {
+            owner: eventData.owner
+            , filters: eventData.subscriptionFilters
+        };
+
+        if (raw_event_message) { meta = Object.assign({}, meta, eventMetadata); }
```

Comment:
Looks like (effectively) there's 2 `if (raw_event_message)` checks here and above in the `const line` assignment. Combine them to save a check?

Reply:
Done!